test/NHSD.BuyingCatalogue.Documents.API.Integration.Tests/Features/Get Solution Documents.3657.feature

```diff
@@ -15,7 +15,7 @@ Background:
 		| sol5       | File with whitespace in its name |
 		| sol6       | děkuji.pdf                       |
 		| sol6       | Благодаря ти.blin                |
-		| sol6       | ありがとうございました.txt         |
+		| sol6       | ありがとうございました.txt                  |
```

Comment:
Vertical bar closer in line to the others? (Aware it can't be perfect due to non-fixed-width characters).

Reply:
👍